### ISSUE TITLE
Add multilanguage support

### ISSUE THREAD

**Issue by csacanam** [ragunathjawahar/android-saripaar#141]
You could use a strings.xml in order to give support to many languages. I could help you with spanish.

**Comment by ragunathjawahar** [comment]
@csacanam yes, we should be doing aars and localized strings. Feel free to send a PR. The code style is same as in [AOSP](https://source.android.com/source/code-style.html).

**Comment by robinsingh-bw** [comment]
Is there any way to do this without modifying the library (or waiting for a PR)?

**Comment by ragunathjawahar** [comment]
You could use the `messageResId` attribute for localization. This issue is for the default error messages.

**Comment by robinsingh-bw** [comment]
Is it possible to specify a default messageResId for the different validators instead of adding it manually everywhere?

**Comment by ragunathjawahar** [comment]
@robinsingh-bw There is a related issue https://github.com/ragunathjawahar/android-saripaar/issues/92, I am still thinking about the best way to do this. Inputs are welcome.

**Comment by robinsingh-bw** [comment]
It is probably an overkill to force a default messageResId in a lib (which would have been ideal). My suggestion would be something simpler such as: 
     validator.setDefaultMessage(final Class<? extends Annotation> annotation, String message)
Then we can just setup default messages for all validators in a utility class or something.

**Comment by LinkOut** [comment]
Any news about translations?

**Comment by MahmoudAlyuDeen** [comment]
Localized error messages would really complete this already awesome library. I can help with Arabic translation.

**Comment by Isabellle** [comment]
I can help with french translation as well
